sdk/src/test/java/com/horizen/cryptolibprovider/InMemorySparseMerkleTreeWrapperTest.java

```diff
@@ -158,4 +158,37 @@ public void removeLeaves() {
             merkleTreeWrapper.close();
         } catch (Exception ignored) {}
     }
+
+    @Test
+    public void heightCheck() {
+        boolean illegarArgumentExceptionOccured = false;
+        // Negative height check
+        try {
+            new InMemorySparseMerkleTreeWrapper(-1);
+        } catch (IllegalArgumentException ex) {
+            illegarArgumentExceptionOccured = true;
+        }
+
+        assertTrue("IllegalArgumentException expected to be occured ", illegarArgumentExceptionOccured);
+
+        illegarArgumentExceptionOccured = false;
+        // Too large height check
+        try {
+            new InMemorySparseMerkleTreeWrapper(23);
```

Comment:
let's reuse `InMemorySparseMerkleTreeWrapper.MAX_TREE_HEIGHT` value for this UT, instead of hardcoding `23` and `22`.
MAX_TREE_HEIGHT can be `public`

Reply:
Done.